### PR TITLE
Fix accessibility issues.

### PR DESCRIPTION
**Pull request by iclanton**
- Fixes https://github.com/microsoft/rushstack-websites/issues/172
- Fixes https://github.com/microsoft/rushstack-websites/issues/170
- Fixes https://github.com/microsoft/rushstack-websites/issues/169
- Fixes https://github.com/microsoft/rushstack-websites/issues/167
- Fixes https://github.com/microsoft/rushstack-websites/issues/166
- Fixes https://github.com/microsoft/rushstack-websites/issues/168

### CODE REVIEW

File: websites/tsdoc.org/docusaurus.config.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 // @ts-check
 // Note: type annotations allow type checking and IDEs autocompletion
 
-const lightCodeTheme = require('prism-react-renderer/themes/github');
-const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+const lightCodeTheme = require('prism-react-renderer/themes/vsLight');
+const darkCodeTheme = require('prism-react-renderer/themes/vsDark');
```
Comment (octogonz) [inline websites/tsdoc.org/docusaurus.config.js:5]: This changes the background color to be white, leaving a ghosted frame:

image

Can we keep this higher contrast theme but restore the original background color?

File: plugins/theme-rushstack-suite-nav/src/theme/Root/index.module.css [hunk 2]
```diff
@@ -1,5 +1,8 @@
 :root {
-  --rushstack-suitenav-background-color: #555555;
+  --docsearch-muted-color: #707783 !important;
+  --rushstack-suitenav-background-color: #444444;
+  --ifm-footer-background-color: var(--rushstack-suitenav-background-color) !important;
+  --ifm-footer-color: #b1b1b1;
```
Comment (octogonz) [inline plugins/theme-rushstack-suite-nav/src/theme/Root/index.module.css:5]: Can you also deploy Heft and LFX so we can see the effects on those sites? Their skins are substantially different from TSDoc

**Review by octogonz — APPROVED** [review]

File: websites/tsdoc.org/docusaurus.config.js (hunk 1 above)
Comment (iclanton) [reply websites/tsdoc.org/docusaurus.config.js:5]: Unfortunately, no. The highlighting color for types is too light.

Added a border to the box.

File: plugins/theme-rushstack-suite-nav/src/theme/Root/index.module.css (hunk 2 above)
Comment (iclanton) [reply plugins/theme-rushstack-suite-nav/src/theme/Root/index.module.css:5]: Done.

**Pull request merged**